test/parallel/test-console.js

```diff
@@ -171,6 +179,14 @@ assert.strictEqual(strings.shift(),
                    "{ foo: 'bar', inspect: [Function: inspect] }\n");
 assert.ok(strings.shift().includes('foo: [Object]'));
 assert.strictEqual(strings.shift().includes('baz'), false);
+assert.strictEqual(strings.shift(),
+                   "{ foo: 'bar', inspect: [Function: inspect] }\n");
+assert.strictEqual(strings.shift(),
+                   "{ foo: 'bar', inspect: [Function: inspect] }\n");
+assert.strictEqual(strings.shift().includes('foo: { bar: { baz:'), true);
+assert.strictEqual(strings.shift().includes('quux'), true);
+assert.strictEqual(strings.shift().includes('quux: true'), true);
```

Comment:
Please do not use `assert.strictEqual(..., true)`. Just use `assert(...)` (or `assert.ok(...)`) to test for logic values. You can find the documentation [here](https://nodejs.org/api/assert.html).

Reply:
Didn't know, sorry. Thanks!

Reply:
No problem, thanks for going through our (sometimes a little long) process!